### ISSUE TITLE
Gradle build error

### ISSUE THREAD

**Issue by vuhoang2102** [airbnb/paris#81]
After add below line in build.gradle, there was a compile error in Android Studio
`implementation 'com.airbnb.android:paris:1.0.0'`
( If I remove above line, it will be OK )
Can you have a look on it?
Thank you

Here is the log detail
`.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jre7/1.2.30/b38fc19e670ebef931fef3ccdcb94eac2b65bb7d/kotlin-stdlib-jre7-1.2.30.jar: kotlin-stdlib-jre7 is deprecated. Please use kotlin-stdlib-jdk7 instead
`

My env info:
- MAC pro 10.13.6
- Android Studio 3.2
- Java 1.8
- Kotlin 1.2.71
- Gradle 3.2.1

**Comment by ngsilverman** [comment]
@vuhoang2102 should be fixed in the next version which I'm getting ready to release. Stay tuned.

**Comment by vuhoang2102** [comment]
`implementation 'com.airbnb.android:paris:1.1.0'`
It's working fine !
Thank you for your prompt handling.

**Comment by ngsilverman** [comment]
Awesome!

**Comment by diegohdez90** [comment]
One question, this is resolved for use latest version: `1.2.1` with app compat v28 using kotlin using the following build.gradle:

```
implementation 'com.android.support:appcompat-v7:28.0.0'
implementation 'com.android.support.constraint:constraint-layout:1.1.3'
implementation 'com.android.support:design:28.0.0'
implementation 'com.android.support:preference-v7:28.0.0'
implementation 'com.android.support:cardview-v7:28.0.0'
```

because display a large error.
